test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -92,9 +94,47 @@
   [response]
   (-> response :body cheshire/parse-string))
 
-(deftest ^:integration environment-classes-integration-test
+(deftest ^:integration environment-classes-integration-cache-disabled-test
+  (testing "when environment classes cache is disabled for a class request"
+    (bootstrap/with-puppetserver-running-with-config
+     app
+     (-> {:jruby-puppet {:max-active-instances 1}}
+         (bootstrap/load-dev-config-with-overrides)
+         (ks/dissoc-in [:puppetserver
+                        :environment-class-cache-enabled]))
```

Comment:
Added an extra reference to the setting in [2ab8dd](https://github.com/camlow325/puppet-server/commit/2ab8ddf465e466adbd38069f1ab14855f6af0e20). in #930.